packages/compass-connection-import-export/src/hooks/use-export.ts

```diff
@@ -48,7 +47,7 @@ export function useExportConnections(
     open: boolean;
     trackingProps?: Record<string, unknown>;
   },
-  exportConnections = storageExportConnections
+  exportConnections = new ConnectionStorage().exportConnections
```

Comment:
Even though unbound methods kinda work in this case, it creates a risk if you are not sure 100% what the implementation looks like, probably better to change this to take the instance instead of a method?